### PR TITLE
PPNA-4785 Add pytest code coverage report

### PR DESCRIPTION
**Pull request by pulina**

### CODE REVIEW

File: python/coverage/action.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    description: AWS Region
+    default: eu-central-1
+  aws-bucket:
+    required: true
+    description: AWS Bucket
+    default: github-code-coverage
+runs:
+  using: "composite"
+  steps:
+    - uses: unfor19/install-aws-cli-action@v1
```
Comment (mrfijas) [inline python/coverage/action.yaml:29]: I think it should be https://github.com/PiwikPRO/actions/blob/da29c8af59d9103a2deb5e376828f6ca693b8341/coverage/action.yml#L23

File: README.md [hunk 2]
```diff
@@ -139,6 +139,36 @@ Example usage:
...
+      - name: Run tests
+        shell: bash
+        run: |
+          pytest -c ${{ github.workspace }}/test-conf/pytest.ini \
+            --html=artifacts/pytest-html/index.html \
+            --cov-config=${{ github.workspace }}/test-conf/coveragerc.ini \
+            --cov=project \
+            --cov-report=html \
+            "${{ github.workspace }}"
+      - name: Run linters
```
Comment (mrfijas) [inline README.md:163]: they're not linters

**Review by mrfijas — COMMENTED** [review]
main comment is about: `PiwikPRO/actions/s3/upload` instead of code coverage

File: README.md [hunk 3]
```diff
@@ -139,6 +139,36 @@ Example usage:
...
+        uses: PiwikPRO/actions/python/coverage@master
```
[the remark this answers is not in the document]
Comment (MarcinMaslany) [reply README.md:164]: yes, great! that's a reusable abstraction, and not code drying!

**Review by MarcinMaslany — APPROVED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pulina) [reply README.md:164]: Ok there is existing action like this but i does not trust it. So for know i will close this PR and repopen new with better name and all stuff.

**Pull request closed without merging**